src/main/java/org/folio/circulation/domain/LoanAction.java

```diff
@@ -8,7 +8,8 @@
   CHECKED_OUT("checkedout"),
   CHECKED_OUT_THROUGH_OVERRIDE("checkedOutThroughOverride"),
   RECALLREQUESTED("recallrequested"),
-  HOLDREQUESTED("holdrequested");
+  HOLDREQUESTED("holdrequested"),
+  DUE_DATE_CHANGE("dueDateChange");
```

Comment:
Why do we need a new status?

Reply:
It's not new

Reply:
It was previously a magic string and now it is in a more appropriate location. It was used only on change due date.